README.md

```diff
@@ -1,62 +1,37 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+Hello Microverse
 
-> Description the project.
+> This is a project as part of a code review exercise.
```

Comment:
- [ **OPTIONAL** ] Good job introducing a description of your project. What if you update it by getting inspiration from the description in the [project's requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=In%20this%20project,to%20know%20them!). I know you make it better.